src/routes/messages.js

```diff
@@ -23,7 +24,7 @@ export const Messages = observer(function Messages(props) {
         if (channelInitialized && !tower.channels.get(channelId).messagesLoadedOnce) {
             tower.channels.get(channelId).fetchMessages();
         }
-    }, [channelId, channelInitialized, tower.channels, towerId]);
```

Comment:
Why did these dependencies get removed? These are all necessary for this `useEffect`.